deeds-dapp-webapp/src/main/webapp/vue-app/dapp/staticPage/components/page/productTour.vue

```diff
@@ -265,30 +265,14 @@
           </v-col>
         </v-row>
       </v-container>
-      <div class="d-flex justify-end">
+      <div class="d-flex justify-end mb-16 pb-16">
         <v-card
           height="8px"
           width="140px"
           color="secondary no-border-radius"
           flat />
       </div>
     </div>
-    <div class="d-flex flex-column mt-16">
-      <span class="display-3 text-center font-weight-bold mt-8 mb-5">{{ $t('startNow') }}</span>
```

Comment:
make sure you remove unused labels too to avoid leaving bloat.

Reply:
done! thanks for the feedback